androidtv/constants.py

```diff
@@ -55,6 +55,9 @@
 #: Get the running apps for a Fire TV device
 CMD_RUNNING_APPS_FIRETV = "ps | grep u0_a"
 
+#: Get installed apps
+CMD_INSTALLED_APPS = "pm list packages | awk -F : '{print $2}'"
```

Comment:
This doesn't work on my (old) 2nd generation Fire TV stick, it says the `awk` command is not found. Can you please change it to simply `pm list packages` and then parse the results in the function.

Reply:
Sure no problem! Thank you for testing on Fire TV Stick hardware didn't have any to hand.